feat: add optional git cloning (#236)
SeisoLLC/easy_infra: docs/Cloning/index.rst

*******
Cloning
*******

``easy_infra`` allows you to clone git repositories at runtime, based on the configuration options specified below.

> Errors encountered during the cloning process are written within the directory that the repos are downloaded in (defaults to ``/iac``), and if any

Author: ```suggestion
Errors encountered during the cloning process are written to ``/var/log/clone.log``, and if there are any errors they are written to ``/var/log/clone.err.log``.
```

> fatal errors are encountered, a log is written to which can then be centralized for analysis if `Logging <../Logging/index.html>`_ is properly

Author: ```suggestion
If fatal errors are encountered, logged are written to which can that will centralized for analysis if `Logging <../Logging/index.html>`_ is properly
```
